package.json

```diff
@@ -101,21 +101,20 @@
   },
   "devDependencies": {
     "@types/glob": "^7.1.3",
-    "@types/mocha": "^8.0.4",
     "@types/node": "^12.11.7",
     "@types/vscode": "^1.46.0",
     "@typescript-eslint/eslint-plugin": "^4.9.0",
     "@typescript-eslint/parser": "^4.9.0",
+    "@vscode/test-electron": "^2.1.3",
```

Comment:
reference: https://www.npmjs.com/package/vscode-test

Reply:
Hmm... I think this package is never used @.@
Sorry for bothering you...